### ISSUE TITLE
Windows: release Bazel as .msi installer

### ISSUE THREAD

**Issue by laszlocsomor** [bazelbuild/bazel#8813]
## Description of the problem / feature request:

On Windows, release an .msi installer for Bazel along with the .exe and .zip files.

https://github.com/bazelbuild/bazel/pull/8579 added logic to build an .msi package.

We could already build unsigned packages, but Windows Defender and Internet Explorer would warn against downloading / installing these.

To sign the binary and the .msi, there's more to do:
- get certificate to sign Bazel binaries and installers
- add .msi build step to release pipeline

### Feature requests: what underlying problem are you trying to solve with this feature?

Easier installation of Bazel.

The installer can set the PATH envvar and install "bazel.exe" instead of "bazel-&lt;version&gt;-windows-x86_64.exe" so that `bazel build` works. The installer can also handle uninstallation and upgrades. Using an .msi installer is familiar to users.

### What operating system are you running Bazel on?

Windows

**Comment by laszlocsomor** [comment]
Status: on track for Bazel 1.0 -- already added the .msi generating step to the release pipeline on https://github.com/bazelbuild/continuous-integration/tree/testing branch but not yet `master`.

TODO:
- undo https://github.com/bazelbuild/continuous-integration/commit/34c96ef5b98a486492a77b3842d43e2b6d6786da
- merge `testing` to `master` (@philwo: do you plan to do this in the next couple weeks?)

**Comment by dslomov** [comment]
Still blocker for 1.0?

**Comment by philwo** [comment]
@dslomov No, we couldn't get it to work in time and Laszlo is on vacation now - let's ship this later. I'll remove the label.

**Comment by kylecordes** [comment]
I realize there are complexities in getting the MSI going at all, but this seems like a reasonable place for a wish list for future enhancements:

- check for prerequisites, walk the user through installing them
- include buildifier
- include buildozer
- include unused_deps
- include basilisk
- include a few examples, which the user can build to verify all is well
- (optionally?) install basilisk as the bazel binary, pre-seeded with the included Bazel version binaries

In other words, ideally the installer would grow from "install the Bazel tool" to "install Bazel and the first-party tools used with it, plus other files a person may want as they get started”

**Comment by sgowroji** [comment]
Hi there! We're doing a clean up of old issues and will be closing this one. Please reopen if you’d like to discuss anything further. We’ll respond as soon as we have the bandwidth/resources to do so.
